src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -282,6 +282,7 @@ def load_arguments(self, _):
         c.argument('mode', arg_type=get_enum_type([CONST_NODEPOOL_MODE_SYSTEM, CONST_NODEPOOL_MODE_USER]))
         c.argument('max_surge', type=str, validator=validate_max_surge)
         c.argument('labels', nargs='*', validator=validate_nodepool_labels)
+        c.argument('node_taints', type=str, validator=validate_taints)
```

Comment:
There is no need to declare `type=str`, because the default value of `type` is `str`